src/libs/actions/Sidebar.js

```diff
@@ -1,31 +1,24 @@
+import {Keyboard} from 'react-native';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
 /**
  * Hide the sidebar, if it is shown.
  */
 function hide() {
+    Keyboard.dismiss();
     Onyx.set(ONYXKEYS.IS_SIDEBAR_SHOWN, false);
 }
 
 /**
  * Show the sidebar, if it is hidden.
  */
 function show() {
+    Keyboard.dismiss();
     Onyx.set(ONYXKEYS.IS_SIDEBAR_SHOWN, true);
 }
 
-/**
- * Tracks the animating state of the Sidebar.
- *
- * @param {Boolean} isAnimating
- */
-function setIsAnimating(isAnimating) {
-    Onyx.set(ONYXKEYS.IS_SIDEBAR_ANIMATING, isAnimating);
-}
-
```

Comment:
Do we not need this anymore because the animation is embedded with the sidebar screen component?  If so, can we remove the onyx key?

Reply:
I think it was just straight up unused. I didn't see anything using the value of this key (maybe something did at some point) so went ahead and ripped it all out.